backend/src/database/entities/emcr/emcr-personnel.entity.ts

```diff
@@ -39,6 +39,27 @@ export class EmcrPersonnelEntity {
   @Column({ name: 'approved_by_supervisor', type: 'boolean', default: false })
   approvedBySupervisor: boolean;
 
+  @Column({
+    name: 'first_choice_section',
+    type: 'varchar',
```

Comment:
Do we need to update the CHEFS form to include this change for EMCR preferences?

Reply:
Nah, let's not